3p/ampcontext.js

```diff
@@ -266,6 +266,33 @@ export class AbstractAmpContext {
     this.client_.sendMessage(MessageType.NO_CONTENT);
   }
 
+  /**
+   * Function to register a callback on the iframe messaging client.
+   * This is to accept messages of any type, for 3p communication.
+   * Please use the specific functions for context messaging where possible.
+   * @param {string} messageType
+   * @param {function(?JsonObject)} callback
+   */
+  registerCallback(messageType, callback) {
```

Comment:
+ @lannka If you are OK with exposing the `registerCallback` and `sendMessage` methods

Reply:
Talked offline. nvm